data/migrations/20190401173542_users.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("users", table => {
+    table.increments();
+    table
+      .string("username", 128)
+      .notNullable()
+      .unique();
+    table.string("password", 30).notNullable();
+    table.string("first_name", 20);
+    table.string("last_name", 20);
```

Comment:
not a bad idea to also include these, but for a registration form I'd argue that all of these be required